mail_broker_telegram/models/mail_broker_telegram.py

```diff
@@ -0,0 +1,369 @@
+                attachments.append(attachment_data)
+        if len(body) > 0 or attachments:
+            author = self._get_author(chat.broker_id, update)
+            new_message = chat.message_post(
+                body=body,
+                author_id=author._name == "res.partner" and author.id,
+                broker_type="telegram",
+                date=update.message.date.replace(tzinfo=None),
+                # message_id=update.message.message_id,
+                subtype_xmlid="mail.mt_comment",
+                message_type="comment",
+                attachments=attachments,
+            )
+            related_message_id = (
+                update.message.reply_to_message and update.message.reply_to_message.id
+            )
+            if related_message_id:
+                related_message = (
+                    self.env["mail.notification"]
+                    .search(
+                        [
+                            ("broker_channel_id", "=", chat.id),
+                            ("broker_message_id", "=", related_message_id),
+                        ]
+                    )
+                    .mail_message_id
+                )
+                if related_message and related_message.broker_message_id:
+                    new_related_message = (
+                        self.env[related_message.broker_message_id.model]
+                        .browse(related_message.broker_message_id.res_id)
+                        .message_post(
+                            body=body,
+                            author_id=author._name == "res.partner" and author.id,
+                            broker_type="telegram",
+                            date=update.message.date.replace(tzinfo=None),
+                            # message_id=update.message.message_id,
+                            subtype_xmlid="mail.mt_comment",
+                            message_type="comment",
+                            attachments=attachments,
+                        )
+                    )
+                    new_message.broker_message_id = new_related_message
+            return new_message
+
+    async def _send_telegram(
+        self, broker, record, auto_commit=False, raise_exception=False, parse_mode=False
+    ):
+        bot = self._get_telegram_bot(broker.token)
+        await bot.initialize()
+        chat = await bot.get_chat(record.broker_channel_id.token)
+        message = False
+        if record.mail_message_id.body:
+            message = await chat.send_message(
+                html2plaintext(record.mail_message_id.body), parse_mode=parse_mode
+            )
+        for attachment in record.mail_message_id.attachment_ids:
+            if attachment.mimetype.split("/")[0] == "image":
+                new_message = await chat.send_photo(
+                    BytesIO(base64.b64decode(attachment.datas))
+                )
+            else:
+                new_message = await chat.send_document(
```

Comment:
Could you please add comment with link to telegram api documentation, where it says about size limit?